python/pyspark/pandas/frame.py

```diff
@@ -4563,9 +4563,9 @@ def shift(self, periods: int = 1, fill_value: Optional[Any] = None) -> "DataFram
         Shift DataFrame by desired number of periods.
 
         .. note:: the current implementation of shift uses Spark's Window without
-            specifying partition specification. This leads to move all data into
-            single partition in single machine and could cause serious
-            performance degradation. Avoid this method against very large dataset.
+            specifying partition specification. This leads to moving all data into
+            a single partition in a single machine and could cause serious
```

Comment:
This leads to moving all data into a single partition in a single machine and could cause serious performance degradation. Avoid this method against very large datasets.
or
This leads to move all data into a single partition in a single machine and could cause serious performance degradation. Avoid this method against very large datasets.

Reply:
The first one, "leads to moving"
I would also change "against very large" to "with very large"